packages/lambda-tiler/src/routes/tile.style.json.ts

```diff
@@ -268,6 +268,9 @@ export async function styleJsonGet(req: LambdaHttpRequest<StyleGet>): Promise<La
     styleConfig.style.layers.filter((f) => !excluded.has(f.id.toLowerCase())),
   );
 
+  // Ensure elevation for style json config
```

Comment:
could we add a TODO to remove this and relate it to the configuration ticket that fixes it?

Reply:
Fixed. Thank you.